app/views/i_belong_mailer/auto_enrolled.text.erb

```diff
@@ -1,25 +1,26 @@
 
-As such, we've enrolled you on the programme, giving you access to a wider range of resources designed to help your school achieve your goals. If you don't want to continue the programme at this time, let us know, and we will update our records. You can still re-enrol yourself in the future on our website (https://teachcomputing.org/i-belong). 
+As such, we've enrolled you on the programme, giving you access to a wider range of resources designed to help your school achieve your goals. If you don't want to continue the programme at this time, let us know, and we will update our records. You can still re-enrol yourself in the future on our website (<%= about_i_belong_url %>).
 
 Unenrol me from the programme (<%= unenroll__i_belong_auto_enrolment_url %>)
 
 Next steps
-Did you know that completing the Encouraging girls into GCSE computer science course means you're already on the way to achieving the I Belong certificate for your school? 
+Did you know that completing the Encouraging girls into GCSE computer science course means you're already on the way to achieving the I Belong certificate for your school?
 
-Now you're enrolled, you will be able to access your personal dashboard, where you can find out more about the certificate and the next steps required to achieve it.  
+Now you're enrolled, you will be able to access your personal dashboard, where you can find out more about the certificate and the next steps required to achieve it.
 
 Explore activities in your dashboard (<%= i_belong_url %>)
 
-Has your school already completed some of the activities required to achieve the certificate? Tick them off by submitting evidence in your dashboard.  
- 
-Ideas to get you started 
-•	Maximise our recommended key stage 3 curriculum resources, aligned to evidence-based approaches supporting girls' engagement in computer science. Examples include Clear messaging in digital media (https://teachcomputing.org/curriculum/key-stage-3/clear-messaging-in-digital-media), Developing for the Web, and Media - Animations (https://teachcomputing.org/curriculum/key-stage-3/media-animations). 
-•	Provide access for students to a computing-related STEM lunchtime or after-school club and show how your approach is gender responsive. You and your colleagues may be doing this already and be ready to submit your evidence.  
- 
-Need more support?  
- 
-Guidance is available in the programme handbook (<%= static_asset_url('I+Belong+Handbook.pdf') %>) to help you and your colleagues choose meaningful activities for your school. 
+Has your school already completed some of the activities required to achieve the certificate? Tick them off by submitting evidence (<%= static_asset_url('Structuring+your+evidence+for+I+Belong.docx') %>)in your dashboard.
```

Comment:
Another to move to helper method rather than static_asset, will link to the new file in media from before